person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = generate_id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
- [ ] What you have is `of_age?`, not `is_of_age?`. Kindly fix that.

Reply:
I had the `Is_of_age` but I'm getting an error from the linters saying I have to use `of_age`  instead so how do I go on about this. 😄